lib/vquic/ngtcp2.c

```diff
@@ -395,7 +409,117 @@ static int quic_init_ssl(struct quicsocket *qs)
   gnutls_server_name_set(qs->ssl, GNUTLS_NAME_DNS, hostname, strlen(hostname));
   return 0;
 }
+#elif defined(USE_WOLFSSL)
+
+static void wolfssl_logging(const int logLevel, const char *const logMessage)
+{
+    fprintf(stderr, " ** wolfssl[%d]: %s\n", logLevel, logMessage);
```

Comment:
Is there a specific reason to use libc `fprintf` instead of a curl-internal printf-pedant, such as `infof`?

Reply:
That logging callback doesn't seem to have a custom pointer coming in so it will be hard to use `infof()` there.

But: the logging is `#if 0`'ed below so it seems this function should be as well? Or maybe even turn ifdef them on DEBUG_WOLFSSL or something to make it easier to switch on the debugging in a future build?

Reply:
I today regard this as an experiment. And am in favour of removing this entirely. If ever wolfSSL gets a "real" logging callback with a customer pointer, *then* I think this should be connected here.

But as things are, it is better removed.

Reply:
Yeah, we can just as well remove that logging function. I'll do it when I merge.